arviz/tests/test_plots.py

```diff
@@ -32,12 +33,27 @@ class Models:
 
 
 @pytest.fixture(scope='function', autouse=True)
-def clean_plots(request):
+def clean_plots(request, save_figs):
+    """Close plots after each test, optionally save if --save is specified during test invocation"""
     def fin():
+        if save_figs is not None:
+
+            # Retry save three times
```

Comment:
@ahartikainen Added plot save retry based on your previous comment. Does this implementation meet the ask?